docs/cloud-tutorials/database-management.md

```diff
@@ -4,37 +4,43 @@ title: Cloud Database Management
 description: Learn how to manage DBOS Cloud database instances
 ---
 
-In this guide, you'll learn how to manage database instances in DBOS Cloud.
-
 ### Provisioning Database Instances
 
-Before you can deploy an application to DBOS Cloud, you must provision a Postgres database instance for it.
+Before you can deploy an application to DBOS Cloud, you must provision a Postgres database instance (server) for it.
 You must choose a database instance name, username and password.
-Both the database instance name and username must be 3 to 16 characters long and contain only lowercase letters, numbers and underscores.
-The database password must contain 8 or more characters.
+
+:::info
+* Both the database instance name and username must be 3 to 16 characters long and contain only lowercase letters, numbers and underscores.
+* The username must start with a letter.
+* The usernames `dbosadmin`, `dbos`, `postgres` and `admin` are reserved and cannot be used.
```

Comment:
Good to know :sweat_smile: This is currently not handled in the console I think.

Reply:
Let's fix it ;-)